### PR TITLE
Add info on how to disable deprecation warnings.

### PR DESCRIPTION
**Pull request by markstory**
Not everyone has the time to fix all the deprecation warnings right after upgrading.

Refs cakephp/cakephp#7887

### CODE REVIEW

File: en/appendices/3-2-migration-guide.rst [hunk 1]
```diff
@@ -8,8 +8,24 @@ Minimum PHP 5.5 Required
...
+Date and Time libraries and remove dependencies on password compatibility
+libraries.
+
+Disabling Deprecation Warnings
+==============================
+
+Upon upgrading you may encounter several deprecation warnings. These warnings
+are emitted by methods, options and functionality that will be removed in
+CakePHP 4.x, but will continue to exist throughout the lifetime of 3.x. While we
+recommend addressing deprecation issues as they are encountered, but that is not
```
Comment (antograssiot) [inline en/appendices/3-2-migration-guide.rst]: extra "but" maybe ?

Comment (markstory) [inline en/appendices/3-2-migration-guide.rst]: Thanks, I missed that.

File: en/appendices/3-2-migration-guide.rst [hunk 2]
```diff
@@ -60,7 +76,14 @@ a new ``CorsBuilder`` has been added. This class lets you define CORS related
...
 
 * Containing the same association multiple times now works as expected, and the
   query builder functions are now stacked.
+
+
+Shell
+=====
+
+* ``Shell::info()``, ``Shell::warn()`` and ``Shell::success()`` were added.
+  These helper methods make using commonly used styling simpler.
```
Comment (dereuromark) [inline en/appendices/3-2-migration-guide.rst]: Is it worth noting that warn() goes to stderr instead - compared to the other two.

Comment (markstory) [inline en/appendices/3-2-migration-guide.rst]: I don't think so. That would be in the API docs.

**Comment by dereuromark** [comment]
:+1:

**Pull request merged**